pkg/gds/gds_test.go

```diff
@@ -1,4 +1,4 @@
-package gds_test
+package gds
```

Comment:
This stems from `admin.go` needing to be in the gds package, so that I can inject the test fixture database into the Admin object. It seemed easier than trying to mock a service, but I definitely understand if we're drawing a hard line at separating the test code and real code into different packages, although we aren't following that rule 100% currently (see `pkg/gds/sectigo/creds_test.go`.